appstore/apps_core_services/views.py

```diff
@@ -119,18 +122,44 @@ def get_context_data(self, *args, **kwargs):
         }
 
 
-# class IrodsLogin(generic.TemplateView):
-#     template_name = 'irods_login.html'
-#
-#     def get(self, *args, **kwargs):
-#         #session = helpers.make_session()
-#         #user = session.users.get('aniroo94')
-#         #print("===>" ,session.query(user.name).first())
-#         return render(self.request, self.template_name)
-#
-#     def post(self, *args, **kwargs):
-#         print(self.request.POST.get("irods_email"))
-#         return render(self.request, self.template_name)
+class IrodsLogin(generic.TemplateView):
+    template_name = 'irods_login.html'
+
+
+    def post(self, *args, **kwargs):
+        email = self.request.POST.get("irods_email")
+        zone = 'NeuroZone'
+        collection = ["/NeuroZone/projects/Top1/raw", "/NeuroZone/projects/Top1/analyzed","/NeuroZone/projects/Top1", "/NeuroZone/projects"  ]
```

Comment:
Can we get these zone and collection names out of the code into a variable in settings?  I know these won't change much but in case more directories are added in the future or deleted/renamed.

Reply:
Sure I can variables for that.